Pipfile

```diff
@@ -0,0 +1,22 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+dj-database-url = "*"
```

Comment:
You could also use `pipenv install --skip-lock` while generating Pipfile so that '*' wildcard deps are avoided.
Since, this was a concern for some poeple on the thread.
@han-ish @salmanulfarzy